src/_css/main.css

```diff
@@ -251,6 +251,8 @@ code {
   tab-size: 2;
   background: rgba(0, 0, 0, 0.07);
   overflow-wrap: break-word;
+  /* The below is still required for Safari */
+  word-wrap: break-word;
```

Comment:
It shouldn't be necessary [in theory](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#wikiArticle:~:text=The%20property%20was%20originally%20a%20nonstandard,overflow%2Dwrap%2C%20with%20word%2Dwrap%20being%20an%20alias.), but is in practice.